.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
+node_modules/
+cookies
+screenshots
+*_report
+backstop.temp.json
+site-config.json
+sample.txt
```

Comment:
Do we use this?

Reply:
Yep we do. https://github.com/compucorp/cases-product-suite-backstopjs/blob/d5d32a55ba6ef4883a26f4ef4077c09fb6636e0a/gulp-tasks/data-setup-steps/create-sample-upload-file.service.js#L7